openreview/workflows/process/committee_recruitment_request_process.py

```diff
@@ -82,16 +82,16 @@ def process(client, edit, invitation):
             if 'profile_not_found' not in recruitment_status['errors']:
                 recruitment_status['errors']['profile_not_found'] = []
             recruitment_status['errors']['profile_not_found'].append(email)
-        elif invited_group_ids:
-            invited_group_id=invited_group_ids[0]
-            if invited_group_id not in recruitment_status['already_invited']:
-                recruitment_status['already_invited'][invited_group_id] = [] 
-            recruitment_status['already_invited'][invited_group_id].append(email)
         elif member_group_ids:
             member_group_id = member_group_ids[0]
             if member_group_id not in recruitment_status['already_member']:
                 recruitment_status['already_member'][member_group_id] = []
             recruitment_status['already_member'][member_group_id].append(email)
+        elif invited_group_ids:
```

Comment:
I am checking if the user is in the Reviewers group before checking the Reviewers/Invited group. Since this is an else if, we were never getting to the Reviewers group check